src/musikcube/data/locales/uk_UA.json

```diff
@@ -33,7 +33,7 @@
     "browse_title_directory_tracks": "пісні в %s",
     "browse_title_genres": "жанри",
     "browse_title_playlists": "грайлисти",
-    "browse_title_tracks": "пісні (сортувати за %s)",
+    "browse_title_tracks": "пісні (сортувати за альбомом)",
```

Comment:
This line looks a little suspicious -- I think there's supposed to be a `%s` here. Is there a reason this change was made?

Reply:
Oh sorry, i fixed it in new commits